src/api/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const ROUTES = {
+  USERS: '/users/',
+  ROADMAPS: '/roadmaps/',
```

Comment:
I think it's not `Routes`, because for routing we have other components and const. Routign about navigation in out app. It is path for request so better call them `endpoints`, cause we use it only in url to fetch data